### DISCUSSION TITLE
Newbie question: Weird tone curve near black.

### DISCUSSION THREAD

**Discussion by quantex** [eoyilmaz/displaycal-py3#326] [Q&A]
Hi all. I recently tried using DispCal with my Spyder4 colorimeter instead of the 1st party software that came with the colorimeter (which hasn't been supported/updated for awhile now). Viewing swatches through the ICC profile produced generally seems fine except.... my monitor's near-black response now seems quite out-of-whack. Meaning, dark scenes in photos seem much too bright, both through experience and in side-by-side comparison with my iPad Pro (which I generally consider "accurate enough" for casual use).

I'm wondering if there is a n00b mistake I'm making with DispCal that someone here can help point out. I've attached screenshots of my DispCal config on the "Calibration" and "Profiling" UI tabs. DispCal was ran with the relevant CCSS correction files grabbed from displaycal.net. The resulting ICC profiles were installed as display profiles in MacOS Sonoma.

For further context, this "over-brightened" behavior near black seems consistent across the three monitors I'm using (Dell P2418D, Huawei MateView, and a Samsung OLED panel), but is especially obvious on the OLED panel perhaps because I'm used to its ability to display true black. It's **_not_** a problem I've had previously using DataColor's 1st party software... even if that software produced profiles that were imperfect in other ways.

Any tips/help here highly appreciated. Thanks!

<img width="716" alt="Screenshot 2024-03-12 at 15 10 40" src="https://github.com/eoyilmaz/displaycal-py3/assets/8118553/67076b59-508c-47b2-9557-24da3d049712">
<img width="793" alt="Screenshot 2024-03-12 at 15 11 12" src="https://github.com/eoyilmaz/displaycal-py3/assets/8118553/65141dda-d555-4eda-b30f-690d39e6845b">

**Comment by eoyilmaz** [comment]
Hi @quantex your settings seems to be alright, and very dark colors going out of wack is pretty normal. Here is the tone curve of my Dell UP2516:

![Screenshot 2024-03-20 at 07 31 06](https://github.com/eoyilmaz/displaycal-py3/assets/1786804/942f9365-96d9-4e0b-a63b-18fb64893982)

Just to check can you show me your monitors tone curve.
